### DISCUSSION TITLE
Can I see the architecture or code of tiny_yolov3_str?

### DISCUSSION THREAD

**Discussion by kimdongyeong0** [lava-nc/lava-dl#335] [Q&A] [answered]
I would like to know if it is possible to check the architecture or code of tiny_yolov3_str.

**Comment by gwgknudayanga** [comment, marked as answer]
Hi,
It is available here.
https://github.com/lava-nc/lava-dl/tree/main/src/lava/lib/dl/slayer/object_detection/models
